src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const x = price * days;
```

Comment:
what the x?idk..
change the variable name
```suggestion
  const basePrice = price * days;
```